engine/src/main/java/io/camunda/zeebe/streamprocessor/ProcessingStateMachine.java

```diff
@@ -285,7 +285,6 @@ private void processCommand(final LoggedEvent command) {
     } catch (final UnrecoverableException unrecoverableException) {
       throw unrecoverableException;
     } catch (final Exception e) {
-      LOG.error(ERROR_MESSAGE_PROCESSING_FAILED_SKIP_EVENT, command, metadata, e);
```

Comment:
:x: please add to JavaDoc contract of `RecordProcessor` that implementing classes are responsible for logging errors. This is something that @deepthidevaki  will also have to consider for her implementation

Reply:
I've adjusted the Javadoc and added a contract